packages/sources/expert-car-broker/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@chainlink/expert-car-broker-adapter",
+  "version": "0.0.4",
```

Comment:
```suggestion
  "version": "0.0.1",
```

Reply:
@justinkaseman might want to change the example adapter, because this was generated from the yarn command

Reply:
Added a commit for it